crates/database/src/auction_participants.rs

```diff
@@ -0,0 +1,64 @@
+use {
+    crate::{auction::AuctionId, Address},
+    sqlx::{Connection, PgConnection},
+};
+
+/// Participant of a solver competition for a given auction.
+#[derive(Debug, Clone, PartialEq, sqlx::FromRow)]
+pub struct Participant {
+    pub auction_id: AuctionId,
+    pub participant: Address,
+}
+
+pub async fn insert(
+    ex: &mut PgConnection,
+    participants: Vec<Participant>,
```

Comment:
```suggestion
    participants: &[Participant],
```

Reply:
Fixed in https://github.com/cowprotocol/services/pull/1270